# Generated by Django 5.1.4 on 2025-09-20 10:55

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='ErrorLog',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('timestamp', models.DateTimeField(auto_now_add=True)),
                ('path', models.CharField(blank=True, max_length=500, null=True)),
                ('method', models.CharField(blank=True, max_length=10, null=True)),
                ('user', models.CharField(blank=True, max_length=255, null=True)),
                ('exception_type', models.CharField(max_length=255)),
                ('message', models.TextField()),
                ('traceback', models.TextField()),
            ],
        ),
    ]
